components/content/inspira/ui/orbit/Orbit.vue

```diff
@@ -2,53 +2,39 @@
   <svg v-if="path" class="pointer-events-none absolute inset-0 size-full">
     <circle class="stroke-1 stroke-foreground/20" cx="50%" cy="50%" :r="radius" fill="none" />
   </svg>
-  <div
-    :style="{
-      '--duration': duration,
-      '--radius': radius,
-      '--delay': -delay,
-      '--direction': reverse ? 'reverse' : 'normal',
-    }"
-    :class="cn('absolute flex size-full transform-gpu animate-orbit', props.class)"
-  >
+  <div :class="cn('absolute flex size-full transform-gpu animate-orbit', props.class)">
     <slot />
   </div>
 </template>
 
 <script lang="ts" setup>
 import { cn } from "@/lib/utils";
-
-interface Props {
-  class?: string;
-  reverse?: boolean;
-  duration?: number;
-  delay?: number;
-  radius?: number;
-  path?: boolean;
-}
+import { ORBIT_DIRECTION, type Props } from ".";
 
 const props = withDefaults(defineProps<Props>(), {
-  reverse: false,
+  direction: () => ORBIT_DIRECTION.Normal,
```

Comment:
Can we make the directions clockwise and anti-clockwise. I feel it will be better. What do you think?

Reply:
You mean

clockwise and counter-clokwise  ? 

And so removing the alternate one, people will figure it out that it's animation-direction css related. ANd doesn't make sens for orbit the alternate event it's can be an use case

Reply:
animation-direction is for any type of animation, since this component is specific to orbit design, I feel clockwise and counter-clockwise/anti-clockwise makes more sense in readability terms too.

Reply:
@rahul-vashishtha Is it good like that ? Or do you want the props value to also be clockwise and counterclockwise ?  Is the ORBIT_DIRECTION constant enough ?

Reply:
Yes, this seems good. Prop idea is also seems better, but we can go with this for now. It is easy to understand the intention also.